packages/flutter_tools/test/general.shard/compile_batch_test.dart

```diff
@@ -395,7 +395,12 @@ void main() {
           '--no-link-platform',
           '--packages',
           '.packages',
-          '.dart_tools/flutter_build/generated_main.dart',
+          '--source',
+          '.dart_tools/flutter_build/dart_plugin_registrant.dart',
+          '--source',
+          'package:flutter/src/dart_plugin_registrant.dart',
```

Comment:
is `.dart_tools/flutter_build/dart_plugin_registrant.dart` content different than `package:flutter/src/dart_plugin_registrant.dart`? If different, is it possible to assign a different name?

Reply:
Yep, the contents are different.  `.dart_tools/flutter_build/dart_plugin_registrant.dart` is where the generated code is.  `package:flutter/src/dart_plugin_registrant.dart` points to where the generated code is.

We could change the name of `.dart_tools/flutter_build/dart_plugin_registrant.dart` easily.  Changing `package:flutter/src/dart_plugin_registrant.dart` would require a change in the engine since that is where it is looking for it.

Reply:
ok. sg